### PR TITLE
Properly handle the interface ABI

### PR DESCRIPTION
**Pull request by axic**
See #223.

Depends on https://github.com/ethereum/solc-js/pull/77.

### CODE REVIEW

**Comment by axic** [comment]
This is not a final version. I think the JSON manipulation should be in solc-js.

File: src/app/compiler.js [hunk 1]
```diff
@@ -270,6 +276,58 @@ function Compiler (editor, handleGithubCall) {
...
+
+        // add "payable" to everything
+        // FIXME: only for compiler <0.4.0
+        if (semver.lt(currentVersion, '0.4.0')) {
+          item.payable = true
+        }
+      }
+
+      // FIXME: only for compiler <0.x.y
+      //        0.1.2 from Aug 2015 had it. The code has it since May 2015 (e7931ade)
```
Comment (axic) [inline src/app/compiler.js]: @chriseth do you remember which released version it was added in?

Comment (chriseth) [inline src/app/compiler.js]: I think we started versioning in July 2015.

**Comment by axic** [comment]
Build needs restart once https://github.com/ethereum/solc-js/pull/77 is merged.

**Comment by axic** [comment]
This needs a bit more work.

**Comment by axic** [comment]
Works now.

File: src/universal-dapp.js [hunk 2]
```diff
@@ -598,10 +618,19 @@ UniversalDApp.prototype.getCallButton = function (args) {
     })
   }
 
+  var title
+  if (isConstructor) {
+    title = 'Create'
```
Comment (yann300) [inline src/universal-dapp.js]: Does it make sense to name it (Create Contract), (Deploy Contract) or (Create Instance) ..?
so it does not overlap with a potential `Create` function

Comment (axic) [inline src/universal-dapp.js]: That is only shown on the top for `Create` and never mixed with actual functions. Note that the behaviour of that is unchanged.

**Comment by axic** [comment]
This works locally on my browser but fails on the karma. @yann300 do you have any idea?

**Comment by axic** [comment]
Thank you @yann300!

**Pull request merged**